frontend/src/metabase/core/components/Markdown/Markdown.tsx

```diff
@@ -1,17 +1,39 @@
-import React from "react";
+import React, { ComponentPropsWithRef } from "react";
 import remarkGfm from "remark-gfm";
+import ReactMarkdown from "react-markdown";
 import { MarkdownRoot } from "./Markdown.styled";
 
 const REMARK_PLUGINS = [remarkGfm];
 
-export interface MarkdownProps {
+export interface MarkdownProps
+  extends ComponentPropsWithRef<typeof ReactMarkdown> {
   className?: string;
-  children?: string;
+  disallowHeading?: boolean;
```

Comment:
nit: Could we make the reverse change and by default not allow headings and links? And pass `true` in places where we need